pages/k8s/using-vault.md

```diff
@@ -90,6 +90,17 @@ juju run-action vault/0 authorize-charm token={charm token}
   </p>
 </div>
 
+### Refreshing certificates
+
+Currently, Vault does not auto-reissue certificates. Generated certificates
+will expire in 30 days. To refresh the certificates, you should run the
```

Comment:
The 30 days expiration depends on how the certificates and root CA cert are created.  It can be tweaked on the Vault charm by the `default-ttl` and `max-ttl` config options, and the `max-ttl` action param to either the `generate-root-ca` or `upload-signed-csr` actions.

Reply:
Okay, I can add some instruction for that, thanks

Reply:
Actually, looking at the default values for all those options, it seems like the certs should be valid for a year.  The 30 day limit was [a bug](https://bugs.launchpad.net/vault-charm/+bug/1788945) which should be resolved now.

Reply:
@johnsca let me know if these changes resolve this for now